packages/build/src/core/build.ts

```diff
@@ -413,6 +417,7 @@ const initAndRunBuild = async function ({
   dry,
   mode,
   api,
+  token,
```

Comment:
Trying to learn - why were these missing before or why are these now required?

Reply:
This codebase has a peculiar structure where data is passed down to inward modules by de-structuring one big object received as the first argument, and limiting it to the properties that are needed by the child method. In this case, the `token` property was not used in certain methods, so to make it available we had to pass it around from where it was available to where it was needed. Hopefully that makes sense?